sdk/eventhubs/azure_messaging_eventhubs_checkpointstore_blob/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#![recursion_limit = "128"]
+#![warn(missing_docs)]
```

Comment:
Now that we have a newer rustc baseline, use `expect` instead of `warn` and make sure there's an issue open to document whatever needs it.